qutebrowser/config/configfiles.py

```diff
@@ -340,6 +340,10 @@ def migrate(self) -> None:
                                        r'(?<!{)\{title\}(?!})',
                                        r'{current_title}')
 
+        self._migrate_to_multiple('fonts.tabs',
+                                  ('fonts.tabs.selected',
+                                   'fonts.tabs.unselected'))
```

Comment:
Please also add tests to `test_configfiles.py` (`TestYamlMigrations`).

Reply:
Will do.